pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -198,23 +221,65 @@ def main(args):
                                           merge_with=config_lines(args))
     science_path = os.path.join(parset['rdx']['redux_path'], parset['rdx']['scidir'])
 
+    # Parse the files sort by MJD
+    files = np.array([os.path.join(args.full_rawpath, file) for file in args.files])
+    nfiles = len(files)
+    target = spectrograph.get_meta_value(files[0], 'target')
+    mjds = np.zeros(nfiles)
+    for ifile, file in enumerate(files):
+        mjds[ifile] = spectrograph.get_meta_value(file,'mjd', ignore_bad_header=True, no_fussing=True)
+    files = files[np.argsort(mjds)]
+
     # Calibration Master directory
     if args.master_dir is None:
-        msgs.error("You need to set an Environmental variable MOSFIRE_MASTERS that points at the Master Calibs")
+        msgs.error('You need to set an environment variable MOSFIRE_MASTERS that points at the Master Calibs')
```

Comment:
MOSFIRE_MASTERS should be changed to QL_MASTERS, this caused me to add MOSFIRE_MASTERS when running the dev suite before I realized it was an out of date error message!

Reply:
please go ahead and fix.